interp/src/main.rs

```diff
@@ -24,26 +38,41 @@ pub struct Opts {
     pub component: String,
 
     /// Group to interpret
-    #[structopt(short = "g", long = "group")]
+    /// XX(karen): The user can specify a particular group to interpret,
+    /// assuming the group is in `main` if not specified otherwise.
+    #[structopt(short = "g", long = "group", default_value = "main")]
```

Comment:
In general, I'm not entirely sure that this functionality makes sense? It seems like, long term, we would like to interpret an entire program and that interpreting a single group doesn't necessarily make sense unless one assumes the control is _just_ that group